.github/workflows/int-and-connected-test-run.yml

```diff
@@ -159,6 +159,12 @@ jobs:
           name: code-coverage-report
           path: build/jacocoHtml
           retention-days: 10
+      - name: "Archive junit test reports"
+        uses: actions/upload-artifact@v2
```

Comment:
There are some GitHub actions specifically targeted at uploading test result, have you looked at them? I was thinking about. using them but haven't had a chance to look into them yet.  For example:

https://github.com/marketplace/actions/publish-unit-test-results
https://github.com/marketplace/actions/test-reporter

Reply:
Yeah, I looked and none of them looked great.  I kinda just wanted to raw report files. Honestly, I'd love to just have them be available to browse online but this looks like a decent close second